applications/EigenSolversApplication/custom_solvers/eigensystem_solver.h

```diff
@@ -217,7 +212,7 @@ class EigensystemSolver
             eig.compute(ar, br);
 
             if(eig.info() != Eigen::Success) {
-                std::cout << "EigensystemSolver: Eigen solution was not successful!" << std::endl;
+                KRATOS_WARNING("EigensystemSolver:") << "Eigen solution was not successful!" << std::endl;
```

Comment:
I would either always use KRATOS_WARNING_IF or KRATOS_WARNING. Same with INFO

Reply:
+1

Reply:
Next line there is a break, is better that way, otherwise you need to do again the conditional